resources/assets/components/Tags/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { map } from 'lodash';
+
+class Tags extends React.Component {
+  render() {
+
+    return (
+        <ul className="form-actions -inline">
+          <li><b> Tags </b></li>
+          <br/>
+          <li> <input className="button -secondary" value="Good Photo"/></li>
```

Comment:
One thing that could make sense is having an key-value mapping of machine-name to human-readable name, and then looping over that to render this interface. That will probably help us reduce some boilerplate when rendering an existing reportback's information as well.

```js
const tags = {
  'good_photo': 'Good Photo',
  'good_quote': 'Good Quote',
  'hidden': 'Hide In Gallery',
  'sponsor': 'Good for Sponsor',
};

return (
  <ul className="form-actions -inline">
    {map(tags, (label, value) => <input className="button -secondary" value={label}/>)}
  </ul>
```